core/src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -130,10 +132,12 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                                 "/ownSecurity/signIn",
                                 "/ownSecurity/updatePassword")
                         .permitAll()
+                        .requestMatchers(HttpMethod.GET, "/user/isOnline/{userId}/")
+                        .authenticated()
```

Comment:
.requestMatchers(HttpMethod.GET, "/user/isOnline/{userId}/")
                        .authenticated()
why did you need it here to solve the current task?